LICENSE.md

```diff
@@ -1393,6 +1393,12 @@ License: [The Apache Software License, Version 2.0](http://www.apache.org/licens
 
 ===========================================================================
 
+Mapbox Navigation uses portions of the Android DB.
```

Comment:
Do you somehow regenerate this file after changing some dependencies?

Reply:
https://github.com/mapbox/mapbox-navigation-android/blob/1ee796e344ee0573629b3c4ccb6e8c07ff1fd9b8/Makefile#L64-L67

If not regenerated, a CI job will fail.